### ISSUE TITLE
Risor project logo

### ISSUE THREAD

**Issue by myzie** [risor-io/risor#143]
@luisdavim @applejag @abadfox233 @KnockOutEZ - I'm working on Risor logo options. I'd appreciate any feedback on any of these options. A couple look good to me but I'm curious what other people think.

I can work with the designer(s) to iterate on it once I have a leader picked out.

1. ![1](https://github.com/risor-io/risor/assets/1389638/a979bc74-5015-419d-a373-fc7ca7f8625c)
2. ![2](https://github.com/risor-io/risor/assets/1389638/cd9d2c8b-9290-4c95-91e0-b4d856461799)
3. ![3](https://github.com/risor-io/risor/assets/1389638/bd39d01c-03ea-4b51-a59e-3fd0e769b1d1)
4. ![4](https://github.com/risor-io/risor/assets/1389638/03ca64ab-70c7-4e1d-a5de-78b79e62ccde)
5. ![5](https://github.com/risor-io/risor/assets/1389638/928bc115-91a8-4605-81c8-c8fdd30dc6e4)
6. ![6](https://github.com/risor-io/risor/assets/1389638/11cb3994-fc08-45cf-8e5d-31363e054993)
7. ![7](https://github.com/risor-io/risor/assets/1389638/50f5ee2e-6a80-4757-add3-2a5cf1cd7750)
8. ![8](https://github.com/risor-io/risor/assets/1389638/eb670b25-91d6-4d35-ba23-fa6b672e3752)
9. ![9](https://github.com/risor-io/risor/assets/1389638/9ee39838-530b-418c-839f-573f14534a96)
10. ![10](https://github.com/risor-io/risor/assets/1389638/227e354d-b63d-41c7-9c0e-efa5bbf0fe29)
11. ![11](https://github.com/risor-io/risor/assets/1389638/decd2559-3434-416b-8980-ea42100e3931)
12. ![12](https://github.com/risor-io/risor/assets/1389638/f83a0a03-3caf-4ce1-8800-3883fc71ad76)
13. ![13](https://github.com/risor-io/risor/assets/1389638/5579fa81-770a-4d9e-9ea3-78fcd9dcc44c)

**Comment by KnockOutEZ** [comment]
I think this concept looks good and simple. But the "R" design can be improved I think and also the font style.![293096071-03ca64ab-70c7-4e1d-a5de-78b79e62ccde.png](https://github.com/risor-io/risor/assets/70368615/eca124c5-0518-47c7-a63e-9b377ad0f79d)

**Comment by luisdavim** [comment]
These are pretty good. 
I think I like number 2 the most, but I also like the concept behind number 4.

**Comment by applejag** [comment]
Color wise I think Risor is either deep purple, dark blue (to slightly contrast Go's bright blue), or crimson red; all as an accent and with dark background. Just a gut feeling, you know what I mean? Maybe it's the first impression of the docs speaking..

Regarding the options:

1. No. Too narrow and the thin-ness of the lines combined with the design makes me think it's for a spell checker, not scripting language.
2. Yes. Don't know what it means (does it say "17"?) but it looks good.
3. No. Color scheme wise I feel is not a fit, and the shape is a bit messy. Although, [it does have a nice shape](https://youtu.be/thOifuHs6eY).
4. Maybe. Needs iteration. Not super big fan of cutting the R up. Could maybe be like `{Risor}`, and shorter logo like `{R}`, but that would too generic and bordering too much with the R programming language logo.
5. No. Too complex. Could maybe reduce to just a single ring. Or if it's a stab at LUA's logo (which would be appropriate, as both are very much scripting languages), then making it more like the solar system could be interesting. Tricky to have as icon form though.
6. No, same as 1.
7. Yes on the icon, no on the text. It feels very disconnected from the icon.
8. Yes, same as 2. I like the purple style a little more. Gives of a more "synthesized" look, which I think fits.
9. No on the icon, yes on the text. The icon is too complicated. Simplifying it could result in something very interesting.
10. Ditto.
11. No, too simple. Resembles a smiley too much.
12. No, too simple. Looks like a consulting firm logo. Would the icon be the split "o"? The split "o" could maybe be something to elaborate on.
13. No on the icon, maybe on the text. Icon has too many colors. The floaty/fluid aspect is interesting, so could maybe be simplified to highlight that. The stretched "o" is the Oracle logo, so just changing that to a regular circle could help.

In summary, I like Nr 2/8 the most out of the box, and see the most potential in Nr 7 and 9/10 with modifications.

Follow-up question: any ideas on a mascot?

**Comment by abadfox233** [comment]
I prefer a minimalist logo. I lean towards either option 2 or 8.

**Comment by luisdavim** [comment]
> 4. Maybe. Needs iteration. Not super big fan of cutting the R up. Could maybe be like {Risor}, and shorter logo like {R}, but that would too generic and bordering too much with the R programming language logo.

yeah it might be worth testing some iterations on no 4, I like the idea of `{Risor}` and `{R}`

**Comment by myzie** [comment]
Thanks all for your thoughts.

I agree that 8 is promising, and that iterations on 4 could be good too.

`{Risor}` could be fun. Or `{Ri}`.

@applejag - this does relate to the mascot question for sure. I think it brings up how close to the Go ecosystem we want to consider Risor being, since any reference to something like a Gopher would lend itself to that. I think it could be difficult to make a "brand" that combines the name Risor, with a logo (like 8, for arguments sake), _and_ a mascot. Seems like they probably wouldn't go together well unless someone was a really talented designer and/or put a lot of time into thinking about this. I lean towards not having a mascot consequently, unless someone has a suggestion.

**Comment by myzie** [comment]
Going with #8.
